provider/kubernetes/kubernetes.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/containous/traefik/log"
 	"github.com/containous/traefik/provider"
 	"github.com/containous/traefik/safe"
+	traefikTls "github.com/containous/traefik/tls"
```

Comment:
I think it's more conventional in Go to not camel case import names. I'd suggest `ttls` or `traefiktls` instead.

Reply:
I thought, I'd reuse the prefix used elswhere in this project like in `acme/acme.go`, `cmd/traefik/traefik.go`, `server/server.go` and `types/types.go` but I can't remember why I introduced the prefix in the first place so I will remove it.

Reply:
It's quite possible we used camel case elsewhere. If we do, we should fix those (by a separate PR).